pkg/csi_driver/node.go

```diff
@@ -384,15 +394,18 @@ func (s *nodeServer) shouldPopulateIdentityProvider(pod *corev1.Pod, optInHnwKSA
 	return tokenVolumeInjected && (sidecarVersionSupported || userInput)
 }
 
-func (s *nodeServer) shouldPassDefaultingFlags(pod *corev1.Pod) bool {
-	var sidecarVersionSupported bool
+func (s *nodeServer) shouldPassDefaultingFlags(gcsFuseSidecarImage string) bool {
```

Comment:
Seems like you removed the use of this function. If its no longer needed, please remove the function